README.md

```diff
@@ -24,10 +24,10 @@ Our goal is faster iteration on massive tabular datasets, both for experimentati
 ### Getting Started
 NVTabular is available in the NVIDIA container repository at the following location: http://ngc.nvidia.com/catalog/containers/nvidia:nvtabular.
 
-The beta (0.3) container is currently available. You can pull the container by running the following command:
+The beta (0.4) container is currently available. You can pull the container by running the following command:
```

Comment:
We don't have a 0.4 container yet - we should keep this at 0.3 for the time being
```suggestion
The beta (0.3) container is currently available. You can pull the container by running the following command:
```